### PR TITLE
Ewl 542 create navigation tabs

### PR DESCRIPTION
**Pull request by Xb0X**

### CODE REVIEW

File: src/modules/Tabs/index.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import { azureRadiance, coreLightMinus1, stormGray } from '../../colors';
+import AllMatchesIcon from '../../assets/tabs/all-matches.svg';
+import MatchIcon from '../../assets/tabs/match.svg';
+import UserCommentsIcon from '../../assets/tabs/user-comments.svg';
+import LiveCommentsIcon from '../../assets/tabs/live-comments.svg';
+import { fontFamilies } from '../../typography';
+import Carousel from '../../components/Carousel';
+import * as breakpoints from '../../breakpoints';
+
+const icons = {
```
Comment (davidgarry) [inline src/modules/Tabs/index.js]: Define an enum for icon names

File: src/modules/Tabs/index.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+Tabs.defaultProps = {
+  defaultTab: null,
+};
+
+Tabs.propTypes = {
+  tabs: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string.isRequired,
+      icon: PropTypes.string.isRequired,
```
Comment (davidgarry) [inline src/modules/Tabs/index.js]: icon can be optional + use oneOf with all values of the enum create above

File: src/modules/Tabs/index.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    let itemToSelect = defaultTab;
+    if (!defaultTabExist && tabs && tabs.length) {
+      itemToSelect = tabs[0].key;
+    }
+    this.setState({ itemSelected: itemToSelect });
+  }
+
+  handleClick(key) {
+    const { onItemSelected } = this.props;
+    onItemSelected(key);
```
Comment (davidgarry) [inline src/modules/Tabs/index.js]: onItemSelected && onItemSelected(key);

File: src/modules/Tabs/index.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+Tabs.propTypes = {
+  tabs: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string.isRequired,
+      icon: PropTypes.string.isRequired,
+      key: PropTypes.string.isRequired,
+    })
+  ).isRequired,
+  onItemSelected: PropTypes.func.isRequired,
```
Comment (davidgarry) [inline src/modules/Tabs/index.js]: Make it optional

File: src/modules/Tabs/index.js [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+          ))}
+        </Carousel>
+      </>
+    );
+  }
+}
+
+Tabs.defaultProps = {
+  defaultTab: null,
+};
```
Comment (davidgarry) [inline src/modules/Tabs/index.js]: onItemSelected: null

File: src/modules/Tabs/index.js [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+  background-position: center;
+  background-size: 32px 32px;
+`;
+
+const StyledIcon = styled(BaseIcon)`
+  display: none;
+  background-image: url(${props => props.icon});
+  ${breakpoints.medium(css`
+    display: inline-block;
+  `)};
```
Comment (val3ntin) [inline src/modules/Tabs/index.js]: missing responsive design for 768px:
* icon height/width
* text size
https://eurosport.invisionapp.com/d/main#/console/17267787/360079611/inspect

File: src/components/Carousel/index.js [hunk 7]
```diff
@@ -37,6 +38,12 @@ const StyledSlidesTrack = styled.div`
   margin-right: auto;
   width: auto;
   transition: transform 200ms ease-out;
+  ${props =>
+    props.justifyContent &&
+    css`
+      display: flex;
+      justify-content: space-between;
```
Comment (val3ntin) [inline src/components/Carousel/index.js]: in case of having e.g. 2 tabs... it might look too weird:
Screen Shot 2019-05-14 at 10 08 05 AM

Comment (Xb0X) [reply src/components/Carousel/index.js]: So I use center when navigation is disabled ;)

File: src/components/Carousel/index.js [hunk 8]
```diff
@@ -37,6 +38,12 @@ const StyledSlidesTrack = styled.div`
   margin-right: auto;
   width: auto;
   transition: transform 200ms ease-out;
+  ${props =>
+    props.disableNavigation &&
+    css`
+      display: flex;
+      justify-content: center;
```
Comment (val3ntin) [inline src/components/Carousel/index.js]: now it'll break the watchBar :( 

Screen Shot 2019-05-14 at 12 55 17 PM

Comment (val3ntin) [reply src/components/Carousel/index.js]: maybe add a new property like `alignCenter` for this ? in order to not bring regression

Comment (Xb0X) [reply src/components/Carousel/index.js]: Oh well spotted ;) it's done ;)

**Pull request merged**